### PR TITLE
Updated API

### PR DESCRIPTION
**Pull request by saketh-n**
Makes sure file type is edf and allows you to upload data from file to the backend, added axios to allow for file upload

### CODE REVIEW

**Comment by saketh-n** [comment]
Correction. DID NOT add axios

File: api/api.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 app.config['CORS_HEADERS'] = 'Content-Type'
 
-@app.route('/result')
+@app.route('/result', methods=['POST'])
 @cross_origin()
 def get_result():
+    # TODO: Eventually can get actual data not just filename. Encrypt it
+    file = request.files['file']
+    filename = file.filename
     return {'result': bool(random.getrandbits(1))}
```
Comment (minneker) [inline api/api.py]: no biggie, for this PR, but should add newline at end of file.

File: src/components/main.js [hunk 2]
```diff
@@ -2,27 +2,52 @@ import { useState } from "react";
...
-      .then(data => {
-        setResult(data.result);
-      });
+    if (file) {
+      const data = new FormData();
+      data.append("file", file);
+      data.append("filename", file.name);
+
+      // TODO: Eventually can get actual data not just filename. Encrypt it
+      fetch("http://localhost:5000/result", {
```
Comment (minneker) [inline src/components/main.js]: Maybe extract URL as global variable so we can change as needed. localhost:5000 is good for now!

**Review by minneker — APPROVED** [review]
Looks good! 2 small comments we can incorporate in next PR.

File: src/components/main.js (hunk 2 above)
Comment (saketh-n) [reply src/components/main.js]: Good suggestion!

File: api/api.py (hunk 1 above)
Comment (saketh-n) [reply api/api.py]: sure

File: src/components/main.js (hunk 2 above)
Comment (devyansh) [reply src/components/main.js]: Good suggestion!

**Pull request merged**